openduck-py/openduck_py/utils/speaker_identification.py

```diff
@@ -0,0 +1,144 @@
+import time
+import os
+from typing import Dict
+
+import torch
+import numpy as np
+from scipy.spatial.distance import cdist
+from pyannote.core import Segment
+from pyannote.audio import Pipeline, Inference, Model
+
+
+HF_AUTH_TOKEN = os.getenv("HF_AUTH_TOKEN")
+
+
+def load_pipelines() -> tuple:
+    """
+    Load the diarization and embedding pipelines.
+    """
+    pipeline = Pipeline.from_pretrained(
```

Comment:
maybe add note that this uses the HF cache (assuming thats how this model is cached)?

Reply:
What do you think this note should say and why is it important to leave as a note?